_distro_map.yml

```diff
@@ -164,7 +170,7 @@ openshift-dedicated:
     enterprise-3.11:
       name: '3'
       dir: dedicated/3
-    enterprise-4.16:
+    enterprise-4.17:
```

Comment:
@karanthshashank, will you please confirm that OSD and ROSA will switch to 4.17 concurrent with OCP GA?

Reply:
@kalexand-rh ACK.